app/frontend/src/components/base/BaseDialog.vue

```diff
@@ -2,7 +2,7 @@
   <v-dialog
     :max-width="width"
     persistent
-    v-if="show"
+    v-bind:value="value"
```

Comment:
Can shorthand this to just `:value="value"`

Reply:
I kind of want to leave this as v-bind since it's so prominent in the docs of how to pass a v-model to a component